service/storage_proxy.cc

```diff
@@ -1504,6 +1504,12 @@ class abstract_write_response_handler : public seastar::enable_shared_from_this<
             timeout_cb();
         }
     }
+    void no_targets() {
+        if (!_cl_achieved) {
+            _error = error::FAILURE;
+        }
```

Comment:
Not sure what error should be returned in this case especially as our docs claim that writes with CL=ANY never fail. I guess `write failure` is not worse than any other.

Reply:
I guess with CL=ANY we'll have _cl_achieved = true.

Reply:
Not if all replicas are dead and hints are disabled.

Reply:
Yes. So need to fix the documentation, please post a patch.